### PR TITLE
Add build instructions to Readme

### PR DESCRIPTION
**Pull request by JakobDev**
#286

### CODE REVIEW

**Comment by SquidDev** [comment]
You don't technically need step 3 unless you're planning to write code for CC - if people just want to experiment with the latest version then `./deploy.sh` is sufficient. 

Seeing as it takes 5-10 minutes to run on my machine, it might be worth mentioning this.

File: README.md [hunk 1]
```diff
@@ -24,3 +24,15 @@ While ComputerCraft will no longer be actively developed by Daniel Ratcliffe, yo
...
 
 The pull requests most likely to be accepted are those which fix bugs, simplify code, or make the mod compatible with newer versions of Minecraft.
+
+## Building
+
+To build ComputerCraft, follow these steps:
+
+1. `git clone` this repository onto your machine.
+2. `cd` to it.
+3.  Run `./deploy.sh` on Unix, or `gradlew.bat build` on Windows.
```
Comment (dan200) [inline README.md]: This isn't good advice, as gradlew.bat build does not do the same things as deploy.sh yet

File: README.md [hunk 2]
```diff
@@ -24,3 +24,15 @@ While ComputerCraft will no longer be actively developed by Daniel Ratcliffe, yo
 * It does not create bugs!
 
 The pull requests most likely to be accepted are those which fix bugs, simplify code, or make the mod compatible with newer versions of Minecraft.
+
+## Building
+
+To build ComputerCraft, follow these steps:
+
+1. `git clone` this repository onto your machine.
```
Comment (dan200) [inline README.md]: We don't need to tell the user how to checkout the repository

File: README.md [hunk 3]
```diff
@@ -24,3 +24,15 @@ While ComputerCraft will no longer be actively developed by Daniel Ratcliffe, yo
...
+2. `cd` to it.
```
Comment (dan200) [inline README.md]: We don't need to tell the user how to change directory

**Pull request closed without merging**